File: RemoteBeesFlow/flow/src/main/java/dev/beehive/remotebees/flow/SignUpContext.kt
/***************************************************************************
 * This source file is part of the RemoteBees open source project.         *
 *                                                                         *
 * Licensed under the MIT License. See LICENSE for license information     *
 ***************************************************************************/

package dev.beehive.remotebees.flow

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class SignUpContext (
    val firstName: String,
    val lastName: String,
    val email: String,
    val password: String
): Parcelable

File: RemoteBeesFlow/RemoteBees/SignUpContext.swift
//
//  SignUpContext.swift
//  RemoteBeesFlow
//

import Foundation

public struct SignUpContext: Hashable, Codable {
    let firstName: String
    let lastName: String
    let email: String
    let password: String
    
    public init(firstName: String, lastName: String, email: String, password: String) {
        self.firstName = firstName
        self.lastName = lastName
        self.email = email
        self.password = password
    }
}
